packages/nextra/src/loader.ts

```diff
@@ -186,6 +186,7 @@ const NextraLayout = __nextra_withSSG__(__nextra_withLayout__({
   headings: ${JSON.stringify(headings)},
   hasH1: ${JSON.stringify(hasH1)},
   ${timestamp ? `timestamp: ${timestamp},\n` : ''}
+  ${unstable_flexsearch ? `unstable_flexsearch: ${JSON.stringify(unstable_flexsearch)}` : ''}
```

Comment:
we don't need it since we specify also in `theme.config`, otherwise, this value should be removed from `theme.config` and we should keep only value from nextra config https://github.com/shuding/nextra/blob/core/packages/nextra-theme-docs/src/types.ts#L54

Reply:
We need to pass the option here. Otherwise user will need to configure twice which is confusing.

Reply:
@promer94 yes it's confusing, so let's remove this option from `theme.config`